.gitignore

```
@@ -0,0 +1,4 @@
+/node_modules
+/dist
+/.cashe
```

Нужно переименовать на:
```suggestion
/.cache
```
А то просочились лишние файлы(